data/transactions/logic/eval.go

```diff
@@ -3056,6 +3056,40 @@ var ecdsaVerifyCosts = []int{
 	Secp256r1: 2500,
 }
 
+// ecdsaVerify cheks a signature,
```

Comment:
@jannotti Minor typo - Treat optionally.

```suggestion
// ecdsaVerify checks a signature,
```